render_html_test.go

```diff
@@ -301,8 +347,9 @@ func TestHTMLNoRace(t *testing.T) {
 		Directory: "fixtures/basic",
 	})
 
+	var err error
```

Comment:
Should I move the error check inside of the `HandlerFunc`, or remove the check entirely. For example, will this help:

```go
h := http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
		err = render.HTML(w, http.StatusOK, "hello", "gophers")
		expectNil(t, err)
 	})
```

Should I do the same for other tests that have similar error checks, or just this one?